packages/styles/src/sass/components/card/_decision.scss

```diff
@@ -0,0 +1,127 @@
+@use '../button/index' as *;
+
+.cbp-card-decision {
+  min-width: 312px;
+}
+
+.cbp-card-decision__content {
+  background-color: var(--cbp-color-white);
+  border: 2px solid var(--cbp-color-base-neutral-lighter);
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  border-bottom: 0;
+  padding: var(--cbp-space-5x) var(--cbp-space-4x) var(--cbp-space-4x) var(--cbp-space-4x);
+}
+
+.cbp-card-decision__actions {
+  display: flex;
+
+  & > button,
+  & > a[href] {
+    @extend .cbp-btn;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    height: 44px;
```

Comment:
I missed this the first time through. Let's not add a fixed height; add padding to get the desired height instead.

Reply:
Removed the fixed height from `_button.scss` but that whole thing needs a refactor and added the correct padding under `.cbp-card__actions`